### ISSUE TITLE
How to train this model?

### ISSUE THREAD

**Issue by LLeicq** [abhaydoke09/Bilinear-CNN-TensorFlow#29]
I'm a new learner, and there are four files in folder CORE .Which file should I run first,next and so on...Thx!

**Comment by abhaydoke09** [comment]
@LLeicq 

You should use bcnn_DD_woft.py first and then bcnn_finetuning.py.
bcnn_DD_woft.py first trains the last layer without backpropagating gradients to initial layers. Once the last layer is done training, bcnn_finetuning.py will train all the layers with lower learning rate.
